pkg/pxc/backup/storage/options.go

```diff
@@ -28,45 +27,55 @@ func GetOptionsFromBackup(ctx context.Context, cl client.Client, cluster *api.Pe
 
 func getAzureOptions(ctx context.Context, cl client.Client, backup *api.PerconaXtraDBClusterBackup) (*AzureOptions, error) {
 	secret := new(corev1.Secret)
-	err := cl.Get(ctx, types.NamespacedName{Name: backup.Status.Azure.CredentialsSecret, Namespace: backup.Namespace}, secret)
+	err := cl.Get(ctx, types.NamespacedName{
+		Name:      backup.Status.Azure.CredentialsSecret,
+		Namespace: backup.Namespace,
+	}, secret)
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to get secret")
 	}
 	accountName := string(secret.Data["AZURE_STORAGE_ACCOUNT_NAME"])
 	accountKey := string(secret.Data["AZURE_STORAGE_ACCOUNT_KEY"])
 
-	endpoint := fmt.Sprintf("https://%s.blob.core.windows.net/", accountName)
-	if backup.Status.Azure.Endpoint != "" {
-		endpoint = backup.Status.Azure.Endpoint
-	}
```

Comment:
can't it be empty string now?

Reply:
It will be set `NewAzure` function, if it's empty: https://github.com/percona/percona-xtradb-cluster-operator/blob/0125d850d8825db6efa2c814b93880074e460771/pkg/pxc/backup/storage/storage.go#L201-L203